bigtop.bom

```diff
@@ -110,14 +110,13 @@ bigtop {
   */
   dependencies = [
     zookeeper:['hadoop', 'hbase'],
-    hadoop:['ignite-hadoop', 'hbase', 'crunch', 'pig', 'hive', 'tez', 'sqoop', 'sqoop2',
+    hadoop:['ignite-hadoop', 'hbase', 'crunch', 'hive', 'tez', 'sqoop', 'sqoop2',
       'oozie', 'mahout', 'flume', 'giraph', 'solr', 'spark','spark1',
       'phoenix', 'alluxio', 'kafka', 'ycsb', 'hama', 'zeppelin',
       'tajo', 'apex'
     ],
     hbase:['phoenix','giraph','ycsb','hive'],
-    pig:['datafu', 'oozie'],
-    hive:['oozie', 'pig','zeppelin'],
+    hive:['oozie', 'zeppelin'],
```

Comment:
Line here conflict with the committed patch in BIGTOP-3074. Please rebase.

Reply:
Hi @evans-ye ,
Thanks for your comments.
But it seems that this PR does not conflict with the committed patch in BIGTOP-3074.
The patch in BIGTOP-3074 is merged to branch-1.3 and has not been merged into 'master' yet. 
And this PR is based on master for the reason that Pig doesn't work with Hive-2.0+.

Reply:
PR #396 is for branch-1.3.